src/workers/summarizeAsk.ts

```diff
@@ -31,12 +31,29 @@ const worker = new Worker(
 
       const fullThread = await getThreadContents(threadChannelId)
 
+      console.log(
+        `\n\n#################\n${JSON.stringify(
+          fullThread,
+          null,
+          2
+        )}\n#################\n\n`
+      )
+
       const partThread = JSON.stringify([
-        fullThread[0],
-        // fullThread[1],
+        `Kundens anläggningsnummer (strAnlNr): ${strAnlNr}`,
```

Comment:
Removed "customer card" from summarization context since that causes it to use name, adress and such as part of SQL-parameterization.

Customer Card example:
```
Kundens namn: Test Testberg
Adress: Anläggningsadress 15 A
Fastighetsbeteckning: Testfastighet 1:2
KundNummer: 100395
Anläggningsnummer: 2034052452
```